docs/gadgets/advise/seccomp-profile.md

```diff
@@ -96,116 +96,6 @@ $ kubectl gadget advise seccomp-profile stop jMzhur2dQjZJxDCI
 }
 ```
 
-### Using `kubectl annotate`
```

Comment:
It is a pitty to lost this example, the correct place to put it would be [here](https://github.com/inspektor-gadget/inspektor-gadget/blob/mauricio/merge-gadgets-documentation/docs/crds/gadgets/seccomp.md), but I am not sure.

Reply:
That's probably the correct place but that file is generated and it's not too easy to put this in the [gadget.go](https://github.com/inspektor-gadget/inspektor-gadget/blob/f79ab2c8f9e9c5defc9f6762204fd9924e897149/pkg/gadget-collection/gadgets/advise/seccomp/gadget.go#L72) documentation and get a well formatted output. Not sure what do to, perhaps I can remove that commit from this PR and handle it later on?